wormhole-connect/package.json

```diff
@@ -63,6 +63,7 @@
     "ethers": "^6.4.2",
     "lz-string": "^1.5.0",
     "material-ui-popup-state": "^5.0.4",
+    "poll": "^3.2.2",
```

Comment:
can we just copy this into `utils` and avoid adding more dependencies? it's like 10 loc: https://github.com/kleinfreund/poll/blob/main/src/poll.js

I am speaking from perspective of avoiding future "dependency hell" like we went through this summer.

Reply:
This is hilarious! I was writing our own polling hook, then wanted to just test the API and opted for something ready to use, which turns out being 10 lines 😄  By the way it's been a long time seeing someone using do-while loop in JS.